test/query-object-test.js

```diff
@@ -3,6 +3,7 @@
 var pgrm = require('../index.js')
 var configs = {dbUrl: "postgres://localhost/pgrm-tests"}
 var pgrmWithDefaults = pgrm(configs)
+var pgrmWithCustomKeys = pgrm(Object.assign(configs, {queryTextKey: 'foo', queryValuesKey: 'bar'}))
```

Comment:
@elysion please use different keys as these collide with the table and the column names.